concent_api/core/views.py

```diff
@@ -503,6 +354,18 @@ def handle_send_reject_report_computed_task(request, client_message):
             requestor_public_key = client_public_key,
             status               = ReceiveStatus
         )
+        subtask = update_subtask(
+            subtask_id                  = client_message.cannot_compute_task.task_to_compute.compute_task_def['subtask_id'],
+            requestor_public_key        = client_public_key,
+            state                       = Subtask.SubtaskState.FAILED,
+            reject_report_computed_task = client_message,
+        )
+        store_pending_message(
+            response_type       = PendingResponse.ResponseType.RejectReportComputedTask,
+            client_public_key   = subtask.provider.public_key_bytes,
```

Comment:
The reject should be sent to the provider inside `ForceReportComputedTaskResponse`.

Reply:
We currently don't handle this kind of response, so I guess you are referring to `RejectReportComputedTask` ?

:+1: I wasn't aware of this, we should create separate issue for creating test for this scenario.

Reply:
Same as I answered earlier, we return `ForceReportComputedTaskResponse` with nested `Reject` message
```python
    elif database_message.response_type == PendingResponse.ResponseType.RejectReportComputedTask.name:  # pylint: disable=no-member
        reject_report_computed_task = deserialize_message(database_message.subtask.reject_report_computed_task.data.tobytes())
        mark_message_as_delivered(database_message)
        response_to_client = message.concents.ForceReportComputedTaskResponse(
            reject_report_computed_task = reject_report_computed_task,
        )
        logging.log_receive_message_from_database(
            response_to_client,
            database_message.client.public_key,
            database_message.response_type,
            database_message.queue
        )
        return response_to_client
```

Reply:
Same as above. The response type in receive queue should (obviously) be the same as the type of the response that is returned from the endpoint.